pkg/asset/internal/templates.go

```diff
@@ -174,6 +178,13 @@ spec:
         - --storage-backend=etcd3
         - --tls-cert-file=/etc/kubernetes/secrets/apiserver.crt
         - --tls-private-key-file=/etc/kubernetes/secrets/apiserver.key
+        livenessProbe:
```

Comment:
unfortunately we can't use a liveness probe here - it will kill the flock'ed api-server, which we need to keep running during checkpoint pivots. We may be able to come up with a solution later - but the short-term fix was to not use the liveness probe.